src/main/java/Deadline.java

```diff
@@ -0,0 +1,15 @@
+public class Deadline extends Task{
+    String dueBy;
```

Comment:
The field dueBy would be better to define this field as private to adhere to good encapsulation practices.